task160_FinalTask/src/test/java/automationpractice/com/pages/ProductDetailsPage.java

```diff
@@ -0,0 +1,34 @@
+package automationpractice.com.pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class ProductDetailsPage extends BasePage {
+    @FindBy(id = "wishlist_button")
+    private WebElement ADD_TO_WISHLIST_BUTTON;
+    @FindBy(name = "Submit")
+    private WebElement ADD_TO_CART_BUTTON;
+    @FindBy(css = "#product_reference span")
+    private WebElement PRODUCT_ID;
+    @FindBy(css = "a.fancybox-close")
+    private WebElement ADDED_TO_WISHLIST_POPUP_CLOSE_BUTTON;
+
+    public ProductDetailsPage() {
+        ADD_TO_CART_BUTTON.isDisplayed();
+    }
+
+    public void pressAddToWishlistButton() {
+        clickOn(ADD_TO_WISHLIST_BUTTON);
+        clickOn(ADDED_TO_WISHLIST_POPUP_CLOSE_BUTTON);
+        try {
+            Thread.sleep(1000);
```

Comment:
It make sense to add comment near each Thread.sleep with explanation, why sleep is used